build-tools/scripts/Packaging.mk

```diff
@@ -98,7 +98,9 @@ _BUILD_STATUS_BUNDLE_INCLUDE = \
 	$(shell find . -name 'CMakeCache.txt') \
 	$(shell find . -name 'config.h') \
 	$(shell find . -name '.ninja_log') \
-	$(shell find . -name 'android-*.config.cache')
+	$(shell find . -name 'android-*.config.cache') \
+	$(wildcard tests/logcat-$(CONFIGURATION)-*.txt) \
```

Comment:
This change should not be necessary as of a6eb47aea5b7369e785280e87e08b9a703f9fb6e.  `logcat-*` files won't even exist when the build status .zip is created.